config/examples/BigTreeTech/SKR Mini E3 1.2/Configuration_adv.h

```diff
@@ -2075,7 +2075,7 @@
    * M912 - Clear stepper driver overtemperature pre-warn condition flag.
    * M122 - Report driver parameters (Requires TMC_DEBUG)
    */
-  //#define MONITOR_DRIVER_STATUS
+  #define MONITOR_DRIVER_STATUS
```

Comment:
This is disabled by default due to the amount of support issues.

Reply:
Ok, i will revert this

Reply:
is reverted in current commit